aggregations-and-analytics/11/11_window_top_n.md

```diff
@@ -0,0 +1,44 @@
+# 11 Window Top-N
+
+![Twitter Badge](https://img.shields.io/badge/Flink%20Version-1.13%2B-lightgrey)
+
+> :bulb: This example will show how to calculate the Top 3 suppliers who have the highest sales for every tumbling 5 minutes window.
+
+The source table (`orders`) is backed by the [`faker` connector](https://flink-packages.org/packages/flink-faker), which continuously generates rows in memory based on Java Faker expressions.
+
+In our previous recipes we've shown how you can [aggregate time series data](../01/01_group_by_window_tvf.md) using the `TUMBLE` function and also how you can get continuous [Top-N results](../05/05_top_n.md).
+In this recipe, we'll use a combination of a tumbling window and the `Window Top-N` [feature](https://ci.apache.org/projects/flink/flink-docs-stable/docs/dev/table/sql/queries/window-topn/) to display the top 3 suppliers with the highest sales every 5 minutes. 
```

Comment:
nit: `a combination of ... Window Top-N` isn't this term already the combination?

Reply:
Fair point. Should we just make it

```suggestion
In this recipe, we'll use the `Window Top-N` [feature](https://ci.apache.org/projects/flink/flink-docs-stable/docs/dev/table/sql/queries/window-topn/) to display the top 3 suppliers with the highest sales every 5 minutes. 
```